src/components/orders/list/OrdersList.js

```diff
@@ -28,8 +28,8 @@ function OrdersList(props) {
 				const deleteResult = await OrdersApiService.deleteOrder(orderId)
 
 				if (deleteResult !== undefined && deleteResult.status === 200) {
-					const response = await OrdersApiService.getAllOrders()
-					setOrders(getOrdersReactiveObjectsList(response.data))
+					const fetchedOrders = await OrdersApiService.getAllOrders()
```

Comment:
this code repeats twice, could you extract it to a method and use a method call instead